### DISCUSSION TITLE
Running pathom3 in nbb

### DISCUSSION THREAD

**Discussion by dehli** [babashka/nbb#326] [Q&A] [answered]
:wave: Hello, I'm running into an issue when trying to use `pathom3` and nbb. I'm getting the following error when I pull in: `com.wsscode.pathom3.interface.async.eql`. 

```
----- Error --------------------------------------
Message:  Could not find namespace: clojure.spec.alpha
Could not find namespace: clojure.spec.alpha
```

It's referenced here: https://github.com/wilkerlucio/pathom3/blob/6493bcea0bb0a29b057297ea3e8a370d756554b9/src/main/com/wsscode/pathom3/interface/async/eql.cljc#L3

I am using `nbb.edn` to pull in my deps however I don't see `clojure/spec` in my `nbb-deps` folder. Any ideas? Thanks!

**Comment by borkdude** [comment, marked as answer]
Pathom3 is a complicated library that does work with bb but I don't think the CLJS equivalent will be compatible with SCI even if we added clojure.spec.alpha to nbb.

**Reply by dehli** [reply]
Thanks for your response! Will try to make my code cljc compatible then so I can go down the bb path.
